### ISSUE TITLE
Link failure 

### ISSUE THREAD

**Issue by eagl0037** [eagl0037/accessible-me#1]
Hey Wendy,

The link keeps saying it is failing and will not let me go the the site.

@wendywarren

**Comment by wendywarren** [comment]
Hi @eagl0037,

Sorry, I thought I read in an email you had the link working but now I see this is actually for the accessible me repository.

I notice you have a couple of yml files included. You wouldn't need those to setup Github pages so I think you may have just selected the wrong option in settings.

Can you send me a screenshot of the Pages tab within the repository settings? It would look like the screenshot here:

<img width="1684" alt="Screen Shot 2024-05-23 at 1 55 25 PM" src="https://github.com/eagl0037/accessible-me/assets/12865609/0fa0fd72-4fcd-4fcd-b411-f777d8486fe4">

Thanks,
Wendy

**Comment by eagl0037** [comment]
Yes, I added the yml because it was saying I didn't have the pages and for it as well as no deployments and the yml helps it.  It is now showing the link but makes it make me have a yml most projects  for it to work which I don't understand.
<img width="1512" alt="Screenshot 2024-05-23 at 2 00 57 PM" src="https://github.com/eagl0037/accessible-me/assets/156235610/51b903dd-ec15-4b81-bad4-e397b114ed5c">

**Comment by wendywarren** [comment]
@eagl0037  - Can you try changing the source to 'deploy from branch' instead of Github actions?

**Comment by wendywarren** [comment]
Or were you saying you didn't have that option so you had to set it up this way?

**Comment by eagl0037** [comment]
Yes, whenever I start a new project it says I don't have access to the link and I don't have a deployment and pages its super weird.
________________________________

**Comment by wendywarren** [comment]
Let's look at this together next time you need to enable GH Pages for an upcoming assignment.

**Comment by eagl0037** [comment]
Okay sounds good!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
